python/cudf/cudf/core/column/numerical.py

```diff
@@ -219,7 +220,15 @@ def _binaryop(self, other: ColumnBinaryOperand, op: str) -> ColumnBase:
             if is_bool_dtype(self.dtype) or is_bool_dtype(other):
                 out_dtype = "bool"
 
+        if (
+            op == "__pow__"
+            and is_integer_dtype(self.dtype)
+            and (is_integer(other) or is_integer_dtype(other.dtype))
+        ):
+            op = "__int_pow__"
```

Comment:
This should be `INT_POW` and we should add a second special case here: https://github.com/rapidsai/cudf/blob/b3d1e411f3e32f35260c2c5e7de91226473eea9e/python/cudf/cudf/_lib/binaryop.pyx#L166-L169

Fixing those special cases is out of scope here -- we just don't want to falsely imply that `INT_POW` is a normal Python binary operation by using the `__thing__` style of syntax.
```suggestion
            op = "INT_POW"
```

Reply:
Fixed in c202657cdf3ac3f78b12dd25cc93871f4e79300a.